### ISSUE TITLE
Trip Wire drawing issue.

### ISSUE THREAD

**Issue by joebayles** [Esri/military-symbology#352]
Currently, MIL-STD-2525 and APP-6 disagree on how to draw a trip wire (https://github.com/Esri/joint-military-symbology-xml/issues/525).

Regardless, ArcGIS draws it a third way:
![image](https://user-images.githubusercontent.com/5016557/53607133-6dcaa200-3b8b-11e9-8c1a-b3500d1fe44d.png)

The variable line should be the horizontal line on the "t", not the vertical one.

**Comment by lfunkhouser** [comment]
@joebayles Please update the issue, providing the 2525 and APP6 ways to render as well as the recommended way to implement.

**Comment by joebayles** [comment]
It's in the link in the description, but here you go:

MIL-STD-2525D
![image](https://user-images.githubusercontent.com/5016557/53606549-26dbad00-3b89-11e9-87ea-c8205989d749.png)

APP-6D
![image](https://user-images.githubusercontent.com/5016557/53606561-36f38c80-3b89-11e9-8472-242ddc06f29b.png)

There is a disparity between the draw rules for a trip wire between the two standards. The US Army will investigate which is the proper one, and submit change proposals to one/both standards.

It should also be mentioned that mines in the example are not intended to be a part of the symbol, and are an example. As is the norm with examples vs. templates, it will be removed from the template, and grayed out in the example.

**Comment by csmoore** [comment]
Since there seems to be an error/disagreement in how this should be displayed, is there a recommendation for one of these in the meantime or is this just on hold until that is decided?

If the issue that we are not following the point ordering then that is because I don't think that has been implemented in Pro yet (#255).

**Comment by joebayles** [comment]
@csmore agreed. The committee is going to decide on what it should be, so suggest we hang in the meantime.

The real question is, should we make whatever change we need to make retroactively.

**Comment by jeconley** [comment]
@joebayles was anything decided about this?  With the anchor point drawing that we are currently implementing we have supported for the 2525Bc2/2525Cdrawing with the 3 anchor points.  

From 2525Dc1
_Anchor Points: This symbol requires two anchor points. Points 1 and 2 define the length and orientation of the straight line (trip wire) portion of the symbol.
Size/Shape: Points 1 and 2 determine the length and orientation of the line drawn from the physical mine to the end of the trip wire. The distance between the line connecting points 1 and 2 is the length of the trip wire connected to the mine.
Orientation: Orientation is determined by the anchor points._

I am unclear how to supported the 2 anchor point drawing in 2525D.  How is the length of the feature determined?  Is point 1 supposed to be at the intersection where the two lines cross? 

Also, is the line circled in blue part of the trip wire symbol?  It wasn't included in Bc2/C.  If so how is the length determined?

![image](https://user-images.githubusercontent.com/8083702/118892314-9565cc00-b8b5-11eb-9fe7-9bafd3ad20e0.png)


From Bc2
![image](https://user-images.githubusercontent.com/8083702/118892252-76ffd080-b8b5-11eb-980d-c1e88fc753ee.png)
